projects/elife.yaml

```diff
@@ -2118,3 +2118,19 @@ kubernetes-aws:
                 max-size: 3
                 desired-capacity: 3
             helm: true
+
+large-repo-wrangler:
+    description: when elife-publications requires a large git repository to be wrangled. adhoc instances only.
+    domain: False
+    intdomain: False
+    formula-repo: https://github.com/elifesciences/large-repo-wrangler-formula
+    aws:
+        ec2: {}
+        ext:
+            type: gp2
+            size: 20 # GB
```

Comment:
space is cheap so raise if needed

Reply:
I think 9GB is the largest repo I've had to deal with so far, most are just large enough to cause editorial problems with the github import process